### DISCUSSION TITLE
How to use @enable3d/ammo-physics

### DISCUSSION THREAD

**Discussion by andy008** [enable3d/enable3d#332] [Help / Questions]
I'm importing AmmoPhysics like this:-
import {  AmmoPhysics, ExtendedMesh, PhysicsLoader  } from '@enable3d/ammo-physics';

However,  when I try to instantiate AmmoPhysics:-

this.physics = new AmmoPhysics(scene);

I get:-

ERROR ReferenceError: Ammo is not defined
    at new AmmoPhysics (physics.js:49:33)

**Comment by yandeu** [comment]
Did you use the physicsLoader to load all ammo.js files?
